node/src/network.rs

```diff
@@ -142,14 +146,20 @@ impl MeshNetworkClient {
     /// This is a subset of all_participant_ids, and includes our own participant ID.
     pub fn all_alive_participant_ids(&self) -> Vec<ParticipantId> {
         let mut result = Vec::new();
+        let my_height = *self
+            .get_indexer_heights()
```

Comment:
should grab self.get_indexer_heights() into a variable first to avoid building it repeatedly